src/utils/records.ts

```diff
@@ -280,6 +306,13 @@ export class Records {
       filterCopy.messageTimestamp = messageTimestampFilter;
       delete filterCopy.dateUpdated;
     }
+
+    // contextId conversion to prefix match
+    const contextIdPrefixFilter = contextId ? FilterUtility.constructPrefixFilterAsRangeFilter(contextId) : undefined;
```

Comment:
Nice! Gives the user some flexibility here. Should probably add a comment to the `RecordsFilter` property so the user knows it's a prefix match.

Reply:
Good call again! I have also added a test that proves that we are now capable of querying sub-contexts!